base_riskscore/code/utils.R

```diff
@@ -241,7 +241,7 @@ run_cv_sl_once <- function(seed = 1, Y = NULL, X_mat = NULL,
     SL.library = sl_lib,
     method = method, cvControl = cvControl,
     innerCvControl = innerCvControl,
-    verbose = TRUE
+    verbose = FALSE
```

Comment:
Could you please leave this as `TRUE` so that travis builds complete?

Reply:
Yes, pushing in a bit. (Sorry I misunderstood how we are dealing with Travis; will leave it TRUE from now 👍 )